src/nvim/testdir/bench_re_freeze.in

```diff
@@ -0,0 +1,13 @@
+Test for Benchmarking RE engine
```

Comment:
Shouldn't be too hard to move this to a lua test. We really want to get rid of the `testdir/` directory.